### ISSUE TITLE
InitPanels don't work

### ISSUE THREAD

**Issue by Taroxx** [FrDH/jQuery.mmenu#893]
I upgraded mmenu from 6.0.2 version to latest version and all my dynamic panels don't work.

Chrome console: Uncaught TypeError: Cannot read property 'closest' of undefined;

```javascript
var $menu = $('#menu');
$menu.mmenu({		
			extensions: [
				"fx-menu-slide",
				"pagedim-black",
			],
			searchfield: {
				add: true,
				addTo: '.mm__brands',
				noResults: 'no result',
				placeholder: 'brands'
			},
			"navbars": [
				{
				   "position": "bottom",
				   "content": ["<a class='phone' 8 (000) 0000</a>"]
				}
			]	
        }, {
			searchfield: {clear: true},
      });
       var api = $menu.data('mmenu');


       function initSearchPanel() {

		if (!initSearchPanel.init) {
		    $('#mm-2 .mm-listview .mobile-wrapper').append('<ul class="mobile-search"><input name="search" type="text"></ul>')

		    var $search = $('.mobile-search'),
			  $searchInput = $('input', $search),
			  xhr;

		    api.initPanels($search)
		    initSearchPanel.init = true
		    return $search
		}
		return $('.mobile-search')
	}

        $('body').on('click', '.search__icon', function(e){
		var $search = initSearchPanel()
		api.openPanel($search)
		api.open()
	});    
```

**Comment by FrDH** [comment]
Could be due to some changed classnames or changed logic.
Have you tried logging some of your variables to the console?
```js
console.log( $search );
```

**Comment by Taroxx** [comment]
No, all classnames and all logic did't change. 
I just updated mmenu to latest version, because this version have some very usefull changes for me. 
If I rollback, all work fine.

When evoke `console.log( $search )` before `api.initPanels($search)`, then it will be ok and $search.length is 1
But after initPanels, .length  will be 0 and chrome console displays this:

```
Uncaught TypeError: Cannot read property 'closest' of undefined
    at e.(/anonymous function)._initSearching (devcom/js/libs/mmenu/jquery.mmenu.all.js:3286)
    at HTMLUListElement.<anonymous> (jquery.mmenu.all.js)
    at Function.each (jquery.js:374)
    at n.fn.init.each (jquery.js:139)
    at e.(/anonymous function).<anonymous> (devcom/js/libs/mmenu/jquery.mmenu.all.js:945)
    at e.(/anonymous function).trigger (devcom/js/libs/mmenu/jquery.mmenu.all.js:4922)
    at e.(/anonymous function)._initPanels (devcom/js/libs/mmenu/jquery.mmenu.all.js:7037)
    at e.(/anonymous function).initPanels (devcom/js/libs/mmenu/jquery.mmenu.all.js:1793)
    at Object.n.(/anonymous function) [as initPanels] devcom/js/libs/mmenu/jquery.mmenu.all.js:11482)
    at initSearchPanel (mobile.js:386)
```
mobile.js:386 - `api.initPanels($search)`
Version of Jquery - v2.1.4

I tried to init panel from console in 2 steps, First step I have append a html ellement to panel. Second step I init panel. And it works, but the error described above still remains. 

Ok. I have tried to set `setTimeout()` function to initPanels(), by 0ms. Fun, but init works, but the error still remains. And js handler stop at this step... My goal of the task - open mmenu at current panel by click on html element (like search icon at header or filter icon at another section) at '.mm-page' div. Like this:

```javascript
    $body.on('click', '.search__input', function(e){
		let $search = initSearchPanel()
		api.openPanel($search)
		api.open()
	});
```
But js handler stop at initPanels. On this error.
![image](https://user-images.githubusercontent.com/9588339/40916271-cf3d0efe-6807-11e8-8230-b386baebbcf8.png)

**Comment by Taroxx** [comment]
@FrDH Do You have any Idea, how to fix this bug?

**Comment by FrDH** [comment]
Is there an online example of the issue? A codepen for example?

I had a look at the logic but couldn’t find anything that might cause this issue.

**Comment by Taroxx** [comment]
I sent mail to @frebsite.nl
--


 mail

**Comment by Taroxx** [comment]
The problem was solved in version 7.0.5.
